Controller/FieldController.py

```diff
@@ -1,24 +1,34 @@
 # Under MIT License, see LICENSE.txt
 
-from math import cos, sin, atan2, sqrt
+from math import cos, sin, atan2, sqrt, pi
 
-__author__ = 'RoboCupULaval'
+from Communication import messages_robocup_ssl_geometry_pb2
+from Communication.messages_robocup_ssl_geometry_pb2 import SSL_GeometryFieldSize
 
+__author__ = 'RoboCupULaval'
```

Comment:
Are we still adding these?

Reply:
Probablement un setting dans pycharm que j'ai oublié.